backend/apps/readings/analysis.py

```diff
@@ -32,3 +90,38 @@ def total_view_time(self):
             for view_time in response.get_parsed_views():
                 total_view_time += view_time
         return total_view_time
+
+    @property
+    def question_sentiment_analysis(self):
```

Comment:
should this be a property? I ask because users expect properties to work very fast and this might take a long time.

(I've made the mistake of making things that are slow and have side effects be properties before -- in software that's released and often used and therefore too late to change back, so I want to avoid others making the same mistake I have)

Reply:
And then the "question_text" could be passed in as a variable and it'd be able to be run on any question text.

Reply:
Or does it need to be a property for some reason relating to the Serializer?  (There is a method serializer)

Reply:
> Or does it need to be a property for some reason relating to the Serializer? (There is a method serializer)

I thought these needed to be properties bc of the serializer, but I was totally wrong. The ReadOnlyField will work fine with a method, as long as it has no additional params. My bad!